### PR TITLE
Fix some formatting and add Style guide

### PR DESCRIPTION
**Pull request by polarbub**
The title says it all

### CODE REVIEW

File: GameTick.md [hunk 1]
```diff
@@ -27,8 +28,7 @@ The main loop:
 - If the server is running, check if the server can keep up (<=50mspt), and if it can't (>50mspt) log a warning.
 - Create a new tick monitor
 - Start the tick
-- Call `net.minecraft.server.MinecraftServer.tick`
-- Wait for the next tick
+- Call `net.minecraft.server.MinecraftServer.tick` Wait for the next tick
```
Comment (Dainternetdude) [inline GameTick.md:31]: y

File: Resources/StyleGuide.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+## Images
+- Screenshots that use custom texture packs, UI mods and other custom content should not be used.
+- Only one image should be used for each individual attribute of the article.
+- Images should be dedicated to one attribute; showing parts of bigger contraptions should be avoided.
+- Images should not have the sole purpose of showcasing a bug, instead link to the bug on the [official bug tracker](https://bugs.mojang.com/secure/Dashboard.jspa).
+
+## Coordinates
+- Single in-game coordinates should be capitalized and unspaced ("Y=1" instead of "y=1" or "y = 1").
+- Coordinates should be in the order X, Y, Z with commas delimiting the individual coordinates.
+- Volumes should be in the order X, Y, Z, with each item separated by a multiplication sign ("×"; &times;); "4×3×2" is an area that is 4 blocks wide along the X axis, 3 along the Y axis (vertical), and 2 along the Z axis.
```
Comment (Dainternetdude) [inline Resources/StyleGuide.md:48]: this is not gonna fly imo

File: Resources/StyleGuide.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Commands 
+- Commands should be in code blocks made with backticks \`.
+- Literal keywords that must be typed in chat do not have any brackets for formatting applied (e.g., `/data merge`).
+- A list of valid keywords should be placed in parentheses with each option separated by a pipe (e.g., (get|merge|set|remove).
+- Variables must be inside angle brackets (e.g., `\<target\>`).
```
Comment (Dainternetdude) [inline Resources/StyleGuide.md:54]:
```suggestion
- Variables must be inside angle brackets (e.g., `headYaw`).
```

File: GameTick.md (hunk 1 above)
Comment (polarbub) [reply GameTick.md:31]: Upstream, I can change it if you want

File: Resources/StyleGuide.md (hunk 3 above)
Comment (polarbub) [reply Resources/StyleGuide.md:54]: I have it set up this way becuase this is how Minecraft suggests commands and how most Unix clis write their help pages

File: Resources/StyleGuide.md (hunk 2 above)
Comment (polarbub) [reply Resources/StyleGuide.md:48]: What do you suggest?

**Comment by polarbub** [comment]
Keep in mind the style guide was pulled and condensed from the old wiki's one.

File: Resources/StyleGuide.md (hunk 3 above)
Comment (Dainternetdude) [reply Resources/StyleGuide.md:54]: two lines above that you didn't write `/<command> <argument>`

Comment (polarbub) [reply Resources/StyleGuide.md:54]: That's becuase on line 52 it's a mock article is talking about the data merge command not the data one

**Review by Dainternetdude — APPROVED** [review]

**Pull request merged**